gensim/models/wrappers/ldamallet.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, mallet_path, corpus=None, num_topics=100, alpha=50, id2word=N
             Number of training iterations.
         topic_threshold : float, optional
             Threshold of the probability above which we consider a topic.
+        random_seed: int, optional
+            Random seed to ensure consistent results, default is None   
```

Comment:
No need to write default value in docstring description + `.` at the end of sentence

Reply:
I spelled out the default so that users know that they need not enter a random_seed parameter at all.

Reply:
Default parameters automatically showed in documentation, for this reason, no need to duplicate it in docstring.